db/migrate/20160721103649_add_industry_to_profile.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndustryToProfile < ActiveRecord::Migration
+  def change
+    add_column :profiles, :industry, :string
```

Comment:
Yeah this was supposed to be part of a different pull request. Any way to remove it?